content/en/agent/basic_agent_usage/deb.md

```diff
@@ -24,7 +24,7 @@ This page outlines the basic features of the Datadog Agent for Debian. If you ha
 
 Packages are available for 64-bit x86 and Arm v8 architectures. For other architectures, use the source install.
 
-**Note**: Debian 7 (wheezy) and above is supported. SysVinit is supported in Agent v6.6.0+.
+**Note**: Debian 7 (wheezy) and above is supported in Agent < 6.36.0/7.36.0. Debian 8 (jessie) is supported in Agent >= 6.36.0/7.36.0. SysVinit is supported in Agent v6.6.0+.
```

Comment:
```suggestion
**Note**: Debian 7 (wheezy) is supported until Agent v6.36.0/7.36.0. Debian 8 (jessie) is required since Agent v6.36.0/7.36.0+. SysVinit is supported in Agent v6.6.0+.
```
The previous wording might make it look like Debian 8 wasn't supported before 7.36. Also I think we don't use operators like `<=` in the docs.

Reply:
I copied exactly the wording from https://github.com/DataDog/documentation/pull/12112 - the same PR about SUSE. It'd be good to keep things consistent. However, what's missing here is the `and above` for Debian versions, which is quite important, so I'll add that.